src/envoy/http/authn/http_filter.cc

```diff
@@ -34,7 +34,26 @@ namespace Istio {
 namespace AuthN {
 
 AuthenticationFilter::AuthenticationFilter(const FilterConfig& filter_config)
-    : filter_config_(filter_config) {}
+    : filter_config_(filter_config) {
+  for (const auto& method : filter_config.policy().peers()) {
```

Comment:
can we wrap this entire loop with something that logs every 1000 calls?

Reply:
This is already logs at low frequency. Most users only have 1 method in peers TLS related setting. Also note per @PiotrSikora's comment, I have moved this from the data path to the control path, it should only log when a new authN policy is first initialized and won't trigger for any requests. Is this enough to leave it as-is?

Reply:
Oh, I didn't notice that you moved this to control path. This should be fine, then. Thanks!

Reply:
yeah, SGTM to me too

Reply:
And apparently it wasn't:  https://github.com/istio/istio/issues/11925

Reply:
Yeah, it turns out Envoy actually allocate the filter for every request, so putting it in the constructor doesn't make it on the data path. I moved it to the filter factory `createFilterFactory()` (https://github.com/istio/proxy/pull/2125) and confirmed it's only called when a new config is received by Envoy, not on every request.